docs/source/indepth_tutorial/directed-search.ipynb

```diff
@@ -98,9 +98,9 @@
      "name": "stderr",
```

Comment:
<blockquote>where we are going to search over the levers for solutions that have robust performance over a set of scenarios.</blockquote><p>Explain/define robust performance briefly and link to further resources.</p><p> </p><blockquote>specify our robustness metrics.</blockquote><p>Which robustness metrics are in the workbench? List, link to overview and/or link to each metric.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>There are no robustness metrics in the workbench. Robustness if further specified in mcphail to which I already link.</p><p> </p>